Library/Homebrew/cmd/doctor.rb

```diff
@@ -92,6 +92,28 @@ def check_for_macgpg2
     end
   end
 
+  # Anaconda installs multiple system & brew dupes, including OpenSSL, Python,
+  # sqlite, libpng, Qt, etc. Regularly breaks compile on Vim, MacVim and others.
+  # Is flagged as part of the *-config script checks below, but people seem
+  # to ignore those as warnings rather than extremely likely breakage.
+  def check_for_anaconda
+    return unless which("anaconda")
+    anaconda = which("anaconda").dirname.realpath
+    python = which("python").dirname.realpath
```

Comment:
You can use `conda` without using Anaconda, can't you? I don't know if that case is also bad.

`sys.executable` seems like the right thing to check if Anaconda sets it reasonably.